operator/v1/types_network.go

```diff
@@ -80,6 +80,12 @@ type NetworkSpec struct {
 	// If not specified, sensible defaults will be chosen by OpenShift directly.
 	// Not consumed by all network providers - currently only openshift-sdn.
 	KubeProxyConfig *ProxyConfig `json:"kubeProxyConfig,omitempty"`
+
+	// exportNetworkFlows enables and configures the export of network flows from OVS
+	// by using protocols netFlow, sFlow or ipfix. Only supported on OVN-Kubernetes plugin.
```

Comment:
Will this be supported with different plugins in the future? If not, call it OVNNetworkFlowExport ?

Reply:
Maybe, that is not discarded, thus the generic name. Let's leave as is, it's not specific OVN feature, even if we are just implementing for OVN now.